flask_website/listings/extensions.py

```diff
@@ -480,6 +480,15 @@ def docserver(self):
         github='maxcountryman/flask-seasurf',
         approved=True,
     ),
+    Extension('Flask-PonyWhoosh', 'Jonathan Prieto-Cubides & Felipe Rodriguez',
+        description='''
+            <p>
+              A full-text search engine using Pony ORM and Whoosh.
+        ''',
+        docs='http://pythonhosted.org/flask-ponywhoosh/',
+        github='compiteing/flask-ponywhoosh',
+        approved=True,
```

Comment:
Please don't mark it approved. I unfortunately don't have time to review new extensions with the approval process (which is also outdated). This doesn't really affect the listing except for adding a small star next to it anyway.